src/Search/Search.js

```diff
@@ -19,11 +19,11 @@ class Search extends Component {
       <form className='form'>
         <input type='text' placeholder='Search for a movie title' name='search' value={this.state.searchInput} onChange={e => this.handleChange(e)}/>
         <Link to={`/search/${this.state.searchInput}`}>
-          <button className='search-button' type='submit' name='search' onClick={e => this.props.handleSearch(this.state.searchInput)}>Search Movies</button>
+            <i className='fas fa-search fa-2x' type='submit' name='search' onClick={e => this.props.handleSearch(this.state.searchInput)}></i>
         </Link>
```

Comment:
We should add an aria attribute of role='button' to the `<i>` for accessibility!